apps/studio/components/interfaces/Settings/API/PostgrestConfig.tsx

```diff
@@ -109,15 +109,18 @@ export const PostgrestConfig = () => {
   const isGraphqlExtensionEnabled =
     (extensions ?? []).find((ext) => ext.name === 'pg_graphql')?.installed_version !== null
 
+  const dbSchema = config?.db_schema ? config?.db_schema.replace(/ /g, '').split(',') : []
+  const defaultValues = {
+    dbSchema,
+    maxRows: config?.max_rows,
+    dbExtraSearchPath: config?.db_extra_search_path,
+    dbPool: config?.db_pool,
+  }
```

Comment:
Is `enableDataApi` missing here?

Reply:
it gets set in a useEffect on L158 below cause its a "computed" property on